src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -20,7 +20,7 @@ public class UiManager implements Ui {
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
 
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
-    private static final String ICON_APPLICATION = "/images/address_book_32.png";
+    private static final String ICON_APPLICATION = "/images/clipboard.png";
```

Comment:
```suggestion
    private static final String ICON_APPLICATION = "/images/clipboard.PNG";
```
I think png should be capital letters 🤔